docs/20-integration-guides/10-keycloak/07-create-user-login.md

```diff
@@ -0,0 +1,43 @@
+# Create User and Login
+
+Create user in the CZERTAINLY realm following steps described in the [Creating users](https://www.keycloak.org/docs/latest/server_admin/#proc-creating-user_server_administration_guide) section of the Keycloak documentation, with these attributes:
+- Username: **admin**
+- Email: **admin@czertainly.local**
```

Comment:
in 03-create-realm you are using <CZERTAINLY_DOMAIN> instead of hardcoded example czertainly.local

Reply:
Yes, it is on purpose, email can be different